docs/pvcscale.md

```diff
@@ -0,0 +1,221 @@
+default location for `kubeconfig` file.
+
+### PBENCH_SSH_PRIVATE_KEY_FILE
+Default: `~/.ssh/id_rsa`
+
+Location of ssh private key to authenticate to the pbench results server.
+
+### PBENCH_SSH_PUBLIC_KEY_FILE
+Default: `~/.ssh/id_rsa.pub`
+
+Location of the ssh public key to authenticate to the pbench results server.
+
+### ENABLE_PBENCH_AGENTS
+Default: `false`
+
+Enables/disables the collection of pbench data on the pbench agent Pods. These Pods are deployed by the workloads/tooling.yml playbook
+it needs to be executed independently of PVCScale test.
+
+### PBENCH_SERVER
+Default: `There is no public default.`
+
+DNS address of the pbench results server.
+
+### SCALE_CI_RESULTS_TOKEN
+Default: `There is no public default.`
+
+Future use for pbench and prometheus scraper to place results into git repo that holds results data.
+
+### JOB_COMPLETION_POLL_ATTEMPTS
+Default: `360`
+
+Number of retries for Ansible to poll if the workload job has completed. Poll attempts delay 10s between polls with some additional time taken for each polling action depending on the orchestration host setup.
+
+### PVCSCALE_TEST_PREFIX
+Default: `pvcscale`
+
+Test to prefix the pbench results.
+
+### PVCSCALE_CLEANUP
+Default: `true`
+
+Enables/disables cleanup of this workload on completion. Cleanup in
+PVCScale terminology means deletion of `pvcscale0` project - this operation can be
+resource intensive in case there are many Pods/PVCs  in `pvcscale0` project which are created
+during test.
+
+### PVCSCALE_BASENAME
+Default: `pvcscale`
+
+Basename used by cluster loader for the project(s) it creates.
+
+### PVCSCALE_MAXPODS
+Default: `1`
+
+Maximum number of Pods that will be created by the pvcscale workload. 
+Typically much higher values are used than the default for large scale tests.
+PVCScale does not have implemented mechanism to calculate maximum number of Pods which all allowed to be started on 
+particular cluster, so it is necessary to adapt `PVCSCALE_MAXPODS` to cluster capacity and
+execute it accordingly. 
+Choosing `PVCSCALE_MAXPODS` to be high can lead that many Pods/PVCs are created what can affect cluster limits and peformance.
+
+### PVCSCALE_POD_IMAGE
+Default: `quay.io/openshift-scale/scale-ci-workload`
+
+Pod container image to use for Pods that are pvscale Pods. This image will be used to
+start Pod and bound PVC to it.
+
+### PVCSCALE_STEPSIZE
+Default: `1`
+
+Number of Pods for cluster loader will create before waiting for Pods to become running.
+PVCScale test creates one Pod/PVC at time.
+
+### PVCSCALE_PAUSE
+Default: `0`
+
+Period of time (in seconds) for cluster loader to pause after creating Pods and 
+waiting for them to be running.PVCScale does not wait on Pods to be `READY` state
+before it creates new ones.
+
+### PVCSCALE_STORAGE_SIZE
+Default: `1Gi`
+
+This is default PVC size. If specified as `1Gi` then all Pods will get `1Gi` PVC
+mounted to `/mnt/pvcmount` inside Pod.
+
+### PVCSCALE_STORAGECLASS
+Default: `none`
```

Comment:
The default in the vars file is currently `csi-rdb`, I am not sure which is preferred as the actual default.

Reply:
I am not sure what would be best default here as storage class can be anything, if I leave `csi-rbd` and if not present , test will fail, but that is case for anything if dynamic provisioning is not in place and there is not functional storage class. I expect user to adapt this value prior running test.

Reply:
Got it if we expect the user to change the value as there is no reasonable default lets set this to:

"Default: none"  (Without "`")

And indicate in the description below the "default" choices possible considering the cluster configuration.  Ex: Use `gp2` if your cluster is built on AWS and `csi-rdb` if you have ceph rook installed.